### PR TITLE
Add utilities for scaled Jacobian and condition number

### PR DESCRIPTION
**Pull request by eslickj**
Add some utility functions so you can get the scaled and unscaled Jacobian and condition numbers.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: idaes/core/util/scaling.py [hunk 1]
```diff
@@ -501,6 +501,41 @@ def constraint_autoscale_large_jac(
...
+    jac, jac_scaled, nlp = constraint_autoscale_large_jac(m, no_scale=True)
+    if scaled:
+        return jac_scaled, nlp
+    else:
+        return jac, nlp
+
+
+def jacobian_cond(m, scaled=True, ord=None):
+    """
+    Get the Jacobian matrix at the current model values.
```
Comment (andrewlee94) [inline idaes/core/util/scaling.py:524]: I think this was copied from the previous method and not updated.

**Review by andrewlee94 — CHANGES_REQUESTED** [review]
One minor comment from me.

File: idaes/core/util/scaling.py [hunk 2]
```diff
@@ -521,15 +529,15 @@ def get_jacobian(m, scaled=True):
 
 def jacobian_cond(m, scaled=True, ord=None):
     """
-    Get the Jacobian matrix at the current model values.
+    Get the condion number of the scaled or unscaled Jacobian matirx of a model.
```
Comment (andrewlee94) [inline idaes/core/util/scaling.py:532]: Typos: condition, matrix

**Review by andrewlee94 — CHANGES_REQUESTED** [review]
A couple more typos.

**Review by andrewlee94 — APPROVED** [review]
Looks good to me.

**Comment by eslickj** [comment]
@nareshsusarla, thanks I didn't know about the warning, but it's probably fine.  The matrix inversion will probably take so much longer than anything else that taking some extra time computing the norm is probably nothing.  I'm a bit worried that this will take a really long time or fail for big and/or poorly conditioned problems, but we'll see how it goes.

**Review by nareshsusarla — APPROVED** [review]

**Pull request merged**